test/test_kfac.py

```diff
@@ -1220,3 +1220,124 @@ def test_kfac_does_affect_grad():
     # make sure gradients are unchanged
     for grad_before, p in zip(grads_before, params):
         assert allclose(grad_before, p.grad)
+
+
+def test_save_and_load_state_dict():
+    """Test that KFACLinearOperator can be saved and loaded from state dict."""
+    manual_seed(0)
+    batch_size, D_in, D_out = 4, 3, 2
+    X = rand(batch_size, D_in)
+    y = rand(batch_size, D_out)
+    model = Linear(D_in, D_out)
+
+    params = list(model.parameters())
+    # create and compute KFAC
+    kfac = KFACLinearOperator(
+        model,
+        MSELoss(reduction="sum"),
+        params,
+        [(X, y)],
+        loss_average=None,
+    )
+
+    # save state dict
+    state_dict = kfac.state_dict()
```

Comment:
Question: Should we try to `torch.save` and `torch.load` this `state_dict` to make sure everything works?